src/Elliptic/DiscontinuousGalerkin/DgOperator.hpp

```diff
@@ -802,15 +825,17 @@ struct DgOperatorImpl<System, Linearized, tmpl::list<PrimalFields...>,
           get<Tags::NormalDotFluxForJump<PrimalMortarVars>>(
               remote_data.field_data)));
       primal_boundary_corrections_on_mortar *= penalty;
-      const auto add_remote_avg_contribution = [](auto& lhs, const auto& rhs) {
+      const auto add_avg_contribution = [](auto& lhs, const auto& rhs,
+                                           const double sign = 1.) {
         for (size_t i = 0; i < lhs.size(); ++i) {
-          lhs[i] += 0.5 * rhs[i];
+          lhs[i] += sign * 0.5 * rhs[i];
```

Comment:
[optional] consider precomputing `sign * 0.5` outside the loop. That multiplication is 1/3 the FLOPS in this expression. At least a few years ago, compilers weren't doing that automatically :(